stable/insights-agent/values.yaml

```diff
@@ -71,6 +71,15 @@ installReporter:
       cpu: 100m
       memory: 128Mi
   additionalAnnotations: {}
+  securityContext:
+    allowPrivilegeEscalation: false
+    readOnlyRootFilesystem: true
+    runAsNonRoot: true
+    runAsUser: 1000
```

Comment:
are we missing `privileged: false` here?

Reply:
not anymore, Thanks!